### ISSUE TITLE
Support quality parameter via ImageTagHelper for webp format

### ISSUE THREAD

**Issue by zeto** [contentful/contentful.net#302]
It is possible to set the quality attribute for jpg:

e.g. `<contentful-image jpg-quality="80">`

However this doesn't seem possible with the `webp` format, even though it is supported underneath in the Images API. Can this parameter be exposed by `webp-quality` (or maybe unifiying both with a `quality` attribute) ?

Thank you!

**Comment by Roblinde** [comment]
@zeto thanks for the suggestion. Yes, the attribute is poorly named. It is only a wrapper for the quality attribute so if even if you have a webp image I think it should still work if you set the jpg-quality attribute, but I will make sure to rename it (or expose a new one) in the next release.

**Comment by zeto** [comment]
```
if (JpgQuality.HasValue && isJpg)
{
    queryBuilder.SetJpgQuality(JpgQuality.Value);
}
```

It gets discarded at the moment (thats how I found out) due to not being JPG which makes sense due to the name of the parameter, but maybe it should ignore the format, the images API documentation mentions non 8-bit PNGs. (AVIF maybe also applies ?)

> I will make sure to rename it (or expose a new one) in the next release.

Thank you!

**Comment by Roblinde** [comment]
@zeto ah yes, it's discarded, that's unfortunate. I'll let you know when this is fixed and available.

**Comment by Roblinde** [comment]
@zeto thanks for your patience. This should now be fixed in version 7.2.2
